test/paper-dialog-behavior.html

```diff
@@ -175,10 +184,6 @@ <h2 id="header">Dialog</h2>
               done();
             });
             Polymer.dom(dialog).querySelector('test-buttons').$.dismiss.click();
-            // We don't wait too long to fail.
```

Comment:
they were causing errors on the logs. Anyways it was done not to wait 10 seconds when it fails (over-engineering)